tgui/packages/tgui/backend.ts

```diff
@@ -247,6 +373,31 @@ export const sendAct = (action: string, payload: object = {}) => {
     logger.error(`Payload for act() must be an object, got this:`, payload);
     return;
   }
+  if (Byond.BYOND_MAJOR >= '516') {
```

Comment:
Why only 516?

Reply:
the original code specifies that it shouldn't be used if on trident with a `!Byond.TRIDENT` check here: https://github.com/tgstation/tgstation/pull/90295/files#diff-eda2bffeeb92d85f4980252bee47f489923757fb991132853a474a7c1db9c72aR360

However, we don't have that, so I went with checking the Byond version instead.

In testing, this code bluescreens TGUI windows for large inputs on 515